static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-header-eur-support.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import {
+    getSupporterPaymentRegion as geolocationGetSupporterPaymentRegion,
+    getSync as geolocationGetSync,
+} from 'lib/geolocation';
+import { updateAcquisitionData } from 'common/modules/commercial/acquisitions-ophan';
+
+const componentType = 'ACQUISITIONS_HEADER';
+const campaignId = 'AcquisitionsEurSupport';
+const abTestName = 'AcquisitionsHeaderEurSupport';
+const EURsupportURL = 'https://support.theguardian.com/eu';
+
+type variantName = 'control' | 'support_contribute';
+
+const modifySupportTheGuardianLink = (variant: variantName): void => {
+    [...document.querySelectorAll('.js-change-become-member-link')].forEach(
+        supportTheGuardianLink => {
+            if (!(supportTheGuardianLink instanceof HTMLAnchorElement)) {
+                return;
+            }
+
+            let supportTheGuardianURL = new URL(supportTheGuardianLink.href);
+
+            if (variant === 'support_contribute') {
+                supportTheGuardianURL = new URL(EURsupportURL);
+            }
+
+            const supportTheGuardianUrlWithTestData = updateAcquisitionData(
+                supportTheGuardianURL,
+                {
+                    abTest: {
+                        name: abTestName,
+                        variant,
+                    },
+                }
+            );
+
+            supportTheGuardianLink.href = supportTheGuardianUrlWithTestData.toString();
+        }
+    );
+};
+
+export const acquisitionsHeaderEurSupport: AcquisitionsABTest = {
+    id: abTestName,
+    campaignId,
+    componentType,
+    start: '2018-03-01',
+    expiry: '2018-04-17',
+    author: 'Santiago Villa Fernandez',
+    description:
+        'Points the "support the guardian" link in the header to the eur version of the support site',
+    audience: 1,
+    audienceOffset: 0,
+    successMeasure: 'AV 2.0',
+    audienceCriteria: 'All EUR transaction web traffic.',
+    dataLinkNames: '',
```

Comment:
It looks like [here](https://github.com/guardian/frontend/blob/4d0dc14064e5c3c803db48b2012185638d889f54/static/src/javascripts/__flow__/types/ab-tests.js) that this field is optional, so would it be better to not give it a value rather than declare it as an empty string?

Reply:
fair enough, I will remove it.